File: sdk-py/examples/list_cases.py
from chainwise import ChainWise


def main():
    client = ChainWise()
    cases = client.get_cases(limit=5)

    if not cases:
        print("No published case studies yet.")
        return

    for case in cases:
        print(f"#{case['id']} {case['protocol_name']} — {case['risk_score']}/100 ({case['classification']})")


if __name__ == "__main__":
    main()

File: sdk-js/examples/list-cases.js
const { ChainWise, formatRiskScore } = require('../dist');

async function main() {
  const chainwise = new ChainWise();
  const cases = await chainwise.getCases({ limit: 5 });

  if (cases.length === 0) {
    console.log('No published case studies yet.');
    return;
  }

  for (const c of cases) {
    console.log(`#${c.id} ${c.protocol_name} — ${formatRiskScore(c.risk_score)} (${c.classification})`);
  }
}

main().catch((err) => {
  console.error('Failed to list cases:', err.message);
  process.exit(1);
});
